scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/processor/DefaultIllumina.java

```diff
@@ -298,16 +337,14 @@ public NotificationDto process(File runDirectory, TimeZone tz) throws IOExceptio
                     }
                     Matcher m = FAILED_MESSAGE.matcher(failMessage);
                     // Somehow, scanner will return things that don't match, so, we check again
-                    return m.matches()
-                        ? LocalDateTime.parse(m.group(1), FAILED_MESSAGE_DATE_FORMATTER)
-                        : null;
+                    return m.matches() ? Instant.parse(m.group(1)) : null;
```

Comment:
This is not going to be in the format that is match. You're still going to have to use the `FAILED_MESSAGE_DATE_FORMATTER` to get a `LocalDateTime` and then convert it.

Reply:
Done